btk/metrics.py

```diff
@@ -112,6 +114,37 @@ def meas_ksb_ellipticity(image, additional_params):
     return result
 
 
+def meas_fixed_aperture(image, additional_params):
+    """Utility function to measure flux using fixed circular aperture with sep.
+
+    Args:
+        image (np.array): Image of a single, isolated galaxy with shape (H, W).
+        additional_params (dict): Containing keys 'psf', 'survey' and 'meas_band_num'.
+                                  The psf should be a Galsim PSF model, the survey a btk Survey
+                                  and meas_band_num an integer indicating the band in which the
+                                  measurement is done.
+    """
+    meas_band_num = additional_params["meas_band_num"]
+    band = additional_params["survey"].available_filters[meas_band_num]
+    filt = additional_params["survey"].get_filter(band)
+    verbose = additional_params["verbose"]
+
+    bkg = sep.Background(image[meas_band_num])
+    catalog = sep.extract(image[meas_band_num], 1.5, err=bkg.globalrms)
```

Comment:
If you derive the background RMS from postage stamp I don't think the value will be reliable enough. I think it would be better to take the true value used in the simulation.